src/components/ScrollLock/__tests__/ScrollLock.test.js

```diff
@@ -1,225 +1,180 @@
 import React from 'react'
-import { mount, shallow } from 'enzyme'
-import { ScrollLock } from '../ScrollLock'
+import { cy } from '@helpscout/cyan'
+import ScrollLock from '../ScrollLock'
+import { handleWheelEvent } from '../ScrollLock.utils'
 
 describe('Direction: Y', () => {
   test('Permits scrolling within extent', () => {
-    const wrapper = shallow(
```

Comment:
A lot of these tests had to be refactored since Enzyme's `mount`/`shallow` rendering can't correctly simulate an event (`wheel`) if bound via `addEventListener`.